src/SpectatorView.Unity/Assets/SpectatorView/Scripts/MarkerDetection/ArUco/HoloLens/ArUcoMarkerDetector.cs

```diff
@@ -131,13 +137,15 @@ private void DebugLog(string message)
         /// <inheritdoc/>
         public void StartDetecting()
         {
+            enabled = true;
+
 #if UNITY_WSA
             if (!_detecting)
             {
                 _detecting = true;
                 _markerObservations.Clear();
                 DebugLog("Starting ArUco marker detection");
-                SetupCamera();
+                SetupCameraAsync();

```

Comment:
>SetupCameraAsync(); [](start = 16, length = 19)

I believe this should "await SetupCameraAsync()". If a method returns void and an exception is thrown, it will result in an unhandled exception error, but if an async method returns a Task and that method throws an exception, that Task will become Faulted but the exception won't be rethrown unless someone awaits the task.